packages/edit-site/src/components/global-styles/hooks.js

```diff
@@ -2,6 +2,8 @@
  * External dependencies
  */
 import { get, set, isEqual } from 'lodash';
+import { colord, extend } from 'colord';
+import a11yPlugin from 'colord/plugins/a11y';
```

Comment:
Should we give this a name that connects it to the colord plugin?

Reply:
👍 I'll follow up with this change; auto-merge was on.